C/primitive/elements/jets.h

```diff
@@ -85,19 +85,23 @@ bool output_scripts_hash(frameItem* dst, frameItem src, const txEnv* env);
 bool output_range_proofs_hash(frameItem* dst, frameItem src, const txEnv* env);
 bool output_surjection_proofs_hash(frameItem* dst, frameItem src, const txEnv* env);
 bool outputs_hash(frameItem* dst, frameItem src, const txEnv* env);
+bool output_hash(frameItem* dst, frameItem src, const txEnv* env);
```

Comment:
The code says `output_hash` but the PR description says `output-utxo-hash`. Let's update the latter.